src/components/TeaserFront/Tile.js

```diff
@@ -106,58 +108,60 @@ const Tile = ({
   }
 
   return (
-    <div
-      {...attributes}
-      onClick={onClick}
-      {...colorScheme.set('backgroundColor', 'default')}
-      style={containerStyle}
-      // The styles of the container are defined
-      // on the parent component <TileRow />
-      // with CSS children selectors, they depend
-      // on the column number given as prop.
-      className='tile'
-    >
-      {imageProps && (
-        <div
-          {...(onlyImage ? styles.onlyImageContainer : styles.imageContainer)}
-        >
-          <LazyLoad
-            visible={aboveTheFold}
-            consistentPlaceholder
-            type='span'
-            style={{
-              position: 'relative',
-              fontSize: 0,
-              display: 'inline-block'
-            }}
+    <ColorContextProvider colorSchemeKey={colorSchemeKey}>
```

Comment:
this is a bit suboptimal and may not lead to the desired effects: concretely the `colorScheme.set('backgroundColor', 'default')` just underneath will not actually be based on this context but the one above since `useColorContext` is called beforehand.

Could we move the `ColorContextProvider` to `src/templates/Front/index.js` with a hardcoded `light`? A additional benefit of that approach would also be that it will not have a negative performance effect of repeating a auto context when not necessary (repeating / nested context with a `auto` value may not work as expected in many situations).